code/Test_Definitions/NumberVerification_verify.feature

```diff
@@ -0,0 +1,137 @@
+
+
+@NumberVerification_verify
+Feature: Camara Number Verification API verify
+
+# Input to be provided by the implementation to the tests
+# References to OAS spec schemas refer to schemas specified in
+# https://raw.githubusercontent.com/camaraproject/NumberVerification/main/code/API_definitions/number_verification.yaml
+
+  Background: Common Number Verification verify setup
+    Given the resource "/number-verification/v0"  as  base url
+    And the header "Content-Type" is set to "application/json"
+    And the header "Authorization" is set to a valid access token
+    And the header "x-correlator" is set to a UUID value
+    And the request body is compliant with the schema NumberVerificationRequestBody
+    And the response body is compliant with the schema NumberVerificationMatchResponse
+    And the header "x-correlator" is set to a UUID value
+    And NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 is compliant with the schema DevicePhoneNumber
```

Comment:
Should NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER1 and NUMBERVERIFY_VERIFY_MATCH_PHONENUMBER2 be declared as testing assets or as environment variables at the top of the feature file? This would help testers understand the pre-requisite data required.

Example from [DeviceLocation Retrieval](https://github.com/camaraproject/DeviceLocation/blob/main/code/Test_definitions/Location-Retrieval.feature):
```
  # Input to be provided by the implementation to the tester
  #
  # Implementation indications:
  # * List of device identifier types which are not supported, among: phoneNumber, networkAccessIdentifier, ipv4Address, ipv6Address
  #
  # Testing assets:
  # * A device object which location is known by the network when connected. 2 distinct device are required for some scenario.
  # * A device object identifying a device commercialized by the implementation for which the service is not applicable
  # * A device object which location cannot be provided during test by the network.
```

Example from [OTP Validation](https://github.com/camaraproject/OTPValidation/blob/main/code/Test_Definitions/OTPvalidationAPI.feature#L3):
```
# * api_root: API root of the server URL
# * phone_number: A public identifier (MSISDN) addressing a telephone subscriptionable to receive SMS. Accordingly to E.164 standard, must be prefixed with '+'.
#     *For additional test scenarios, operator could provide phone number for line that cannot receive SMS, line that blocked SMS reception, not belonging to the operator, designing a landline number.
# * message: Message template used to compose the content of the SMS sent to the phone number. It must include the following label indicating where to include the short code {{code}}. Operator could specified a max_lenght for the message.
# * authentication_id: identifier of a a send-code request. Identifier retrieved in send-code response.
# * max_sent: Maximum allowed request to send code for a given device
# * max_try: Maximum allowed try to validate the code for a given autorisation request.
```

Reply:
api_root is never used in the tests. The current feature files submitted are quite different when it come to "environment" and "varialbes". Gherkin does not define a format. Which lead to "everybody does want they think best".
I am looking at https://github.com/cucumber/cucumber-android . Maybe I can steal some ideas from them.
And then run the instrumentation tests on a mobile device (semi-)manually.